src/ir_Mitsubishi.cpp

```diff
@@ -430,11 +615,21 @@ uint8_t IRMitsubishiAC::getMode() {
 void IRMitsubishiAC::setMode(uint8_t mode) {
   // If we get an unexpected mode, default to AUTO.
   switch (mode) {
-    case MITSUBISHI_AC_AUTO: break;
-    case MITSUBISHI_AC_COOL: break;
-    case MITSUBISHI_AC_DRY: break;
-    case MITSUBISHI_AC_HEAT: break;
-    default: mode = MITSUBISHI_AC_AUTO;
+    case MITSUBISHI_AC_AUTO:
```

Comment:
Why do this? Why not change the constants to 0b0011xxxx instead?
Also, Why the extra 0b0011? Did we miss it in the initial decoding analysis?

Reply:
The mode setting wasn't perfectly implemented before. The mode is stored in 2 places of the payload and represented differently. Byte 6 and byte 8. Reference: www.analysir.com/blog/wp-content/uploads/2014/12/Mitsubishi_AC_IR_Signal_Fields.jpg
I checked that my remote controller does exactly this.

Reply:
Ah and D'oh! I missed it was [8] in the switch statement, and [6] afterwards.